src/test/java/com/ays/admin_user/controller/AdminUserRegisterApplicationSystemTest.java

```diff
@@ -276,4 +281,128 @@ void givenValidAdminUserRegisterApplicationId_whenAdminUserApplicationFound_then
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(response.getHttpStatus().name()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(response.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .doesNotExist());
+    }
+
+    @Test
+    void givenPhoneNumberWithAlphanumericCharacter_whenPhoneNumberIsNotValid_thenReturnValidationError() throws Exception {
+
+        // Given
+        String applicationId = AysValidTestData.APPLICATION_ID;
+        AysPhoneNumberRequest phoneNumber = new AysPhoneNumberRequestBuilder()
+                .withCountryCode("ABC")
+                .withLineNumber("ABC").build();
+        AdminUserRegisterApplicationCompleteRequest registerRequest = new AdminUserRegisterRequestBuilder()
+                .withValidFields()
+                .withPhoneNumber(phoneNumber).build();
+
+        // Then
+        String endpoint = BASE_PATH.concat("/registration-application/").concat(applicationId).concat("/complete");
+        AysError errorResponse = AysErrorBuilder.VALIDATION_ERROR;
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .post(endpoint, registerRequest))
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isBadRequest())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(errorResponse.getHttpStatus().name()))
+                .andExpect(AysMockResultMatchersBuilders.header()
+                        .value(errorResponse.getHeader()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(errorResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .doesNotExist())
+                .andExpect(AysMockResultMatchersBuilders.subErrors()
+                        .isNotEmpty());
+    }
+
+    @Test
+    void givenPhoneNumberWithInvalidLength_whenPhoneNumberIsNotValid_thenReturnValidationError() throws Exception {
+
+        // Given
+        String applicationId = AysValidTestData.APPLICATION_ID;
+        AysPhoneNumberRequest phoneNumber = new AysPhoneNumberRequestBuilder()
+                .withCountryCode("456786745645")
+                .withLineNumber("6546467456435548676845321346656654").build();
+        AdminUserRegisterApplicationCompleteRequest registerRequest = new AdminUserRegisterRequestBuilder()
+                .withValidFields()
+                .withPhoneNumber(phoneNumber).build();
```

Comment:
Burada var olanı kullanmak yerine initialize etmemiz gerekiyordu, çünkü var olan üzerinde değişiklik yapıldığında testler çalıştığı sırada tek bir veri tabanı olarak davrandığından testler birbirini engelleyebiliyor

Reply:
@agitrubard @AhmetAksunger Anladığım kadarıyla  `withValidFields`  yerine Postman tarafında kullandığımız request json yapısına göre burayı düzeltmemiz lazım.

Reply:
Düzenledim, kontrol eder misin?

Reply:
Şu an yaptığımız yapıda var olan bir admin user üzerinden yapıyoruz. Aslında sıfırdan bir application oluşturmamız gerekiyor, institution dahil bağımsız bir şekilde olmalı ki testler yazıldığı sırada birbirini ezip hata vermesin. Daha önce bu tarz hatalar aldığımız için testleri buna göre düzenlemiştik. Birlikte bakabiliriz buradaki yapının daha da anlaşılır olabilmesi için.